.github/workflows/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Ignore node_modules
+node_modules
+
+# Ignore .vs folder
+.vs
```

Comment:
- [ ] Your `.gitignore` file should be in the root of your project just as in the  [Template](https://github.com/microverseinc/readme-template) provided. Your `test.md` file is meant to be here. I noticed you have it duplicated. kindly delete this 👍

Reply:
I've added the linters files that were missing, but I still get the same fails in linters evaluation :(